index.js

```diff
@@ -18,17 +29,23 @@ module.exports = (nextConfig = {}) => ({
     }
 
     const {
-      assetPrefix,
-      generateSw = true,
-      dontAutoRegisterSw = false,
       devSwSrc = join(__dirname, 'service-worker.js'),
+      dontAutoRegisterSw = false,
+      generateInDevMode = false,
+      generateSw = true,
+      // Before adjusting "registerSwPrefix" or "scope", read:
+      // https://developers.google.com/web/ilt/pwa/introduction-to-service-worker#registration_and_scope
       registerSwPrefix = '',
       scope = '/',
-      generateInDevMode = false,
-      transformManifest = manifest => manifest,
       workboxOpts = {
-        globPatterns: ['static/**/*'],
-        globDirectory: '.',
+        exclude: preCacheManifestBlacklist,
+        // TODO: Do we want to bundle Workbox inline? Makes the SW + dependencies slightly less cache-able, but it
```

Comment:
@hanford this is probably the most controversial decision for you to weigh in on.

Reply:
I think this is fine 🤷‍♂️